sgp4/tests.py

```diff
@@ -207,6 +207,19 @@ def test_tle_export():
         if satrec.satnum not in expected_errs_line2:
             assertEqual(out_line2, line2)
 
+def test_export_tle_raises_error_for_out_of_range_angles():
+    # See https://github.com/brandon-rhodes/python-sgp4/issues/70
+    sat = Satrec()
```

Comment:
Go ahead and move the creation of `sat` inside the loop, so that folks reading the test later can be assured that there's no state being invisibly carried from one iteration of the loop to the next.

Reply:
Good idea, done!